src/test/compile-fail/issue-17025.rs

```diff
@@ -8,6 +8,8 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// ignore-test the unsized enum no longer compiles
```

Comment:
I have ignored this test for now, but it has effectively become useless and could be deleted but I'm unsure what the policy is in this case.